projects/plugins/jetpack/modules/search/class-jetpack-search-customberg.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * A class that adds a search dashboard to wp-admin.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Search;
+
+use Jetpack_Plan;
+use Jetpack_Search_Helpers;
+
+require_once JETPACK__PLUGIN_DIR . '_inc/lib/admin-pages/class.jetpack-admin-page.php';
+require_once JETPACK__PLUGIN_DIR . '_inc/lib/admin-pages/class-jetpack-redux-state-helper.php';
+
+/**
+ * Responsible for adding a search dashboard to wp-admin.
+ *
+ * @package Automattic\Jetpack\Search
+ */
+class Jetpack_Search_Customberg {
```

Comment:
Do we need two menu entries for `Search`?

Reply:
I thought Customburg would only be linked from the setting pages. Another menu item looks a bit overkill to me.

Reply:
+1 to that 👍

Reply:
This class merely adds a new page at `/wp-admin/admin.php?page=<slug>`, where the slug is `jetpack-search-customize`. See [this line](https://github.com/Automattic/jetpack/pull/20282/files/38d1d3982962ad567fcfbaa91a227bc334f9ac10#diff-119cb8cdd124378b37ca4e8f9df2a855d0208f56e0bb47a986626b61698a7b95R58) prevents a submenu from being added to the wp-admin sidebar.

I'll rename some of the functions in the class to make this distinction clearer.

Reply:
Method names have been updated in 095855b575ce4f87eaa8e4b1cf7201eb65411d78.